app.js

```diff
@@ -1,20 +1,39 @@
-const express = require('express');
-const app = express();
+const express = require("express");
 const port = 3000;
-const path = require('path');
+const path = require("path");
+const bodyParser = require("body-parser");
+const Items = require("./items");
+
+const app = express();
 
-app.use(express.static('public'));
+app.use(express.static(path.join(__dirname, "public")));
 
-const landing_message = 'Application is running successfully. Follow instructions in the README to run the exercises.';
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
 
-app.get('/', (req, res) => res.send(landing_message));
+// All views in vies folder
+app.set("views", path.join(__dirname, "views"));
+
+// Use pug templating
+app.set("view engine", "pug");
+
+app.get("/", (req, res) => {
+	res.render("index", {
+		items: Items
+	});
+});
 
-app.get('/assertions', (req, res) => {
-    res.sendFile(path.join(__dirname + '/src/assertions/index.html')); 
+app.get("/checkout", (req, res) => {
+	const { total } = req.query;
+	res.render("checkout", {
+		total
+	});
 });
 
-app.get('/diagnostic', (req, res) => {
```

Comment:
I should have mentioned, I need this route for something else. Please do not delete it.

Reply:
Okay, fixing it now.